src/main/java/inf112/skeleton/app/Game.java

```diff
@@ -83,27 +83,27 @@ public boolean keyDown(int key) {
     private void executeCard(Card card) {
         switch(card.getKind()) {
             case FORWARD:
-                movePlayer(Linear.scl(playerTransform.direction, card.getSteps()));
+                movePlayer(Linear.scl(robot.transform.direction, card.getSteps())); //endre med robot. transforme. direction
```

Comment:
Skal kommentaren på slutten av linjen være med?

Reply:
Nvm, we fix!